PrivateClinicsWebNet.Application/Services/AppointmentService.cs

```diff
@@ -0,0 +1,127 @@
+﻿using AutoMapper;
+using FluentValidation;
+using PrivateClinicsWebNet.Application.Abstractions;
+using PrivateClinicsWebNet.Application.DTOs.AppointmentDTOs;
+using PrivateClinicsWebNet.Application.Exceptions;
+using PrivateClinicsWebNet.BusinessLogic.Entities;
+using PrivateClinicsWebNet.Infrastructure.AppointmentInfrastructure.Abstractions;
+using PrivateClinicsWebNet.Infrastructure.AppointmentInfrastructure.DTOs;
+using PrivateClinicsWebNet.Infrastructure.Shared.Wrapper;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PrivateClinicsWebNet.Application.Services
+{
+    public class AppointmentService(IAppointmentRepository appointmentRepository,
+        IMapper mapper,
+        IValidator<AppointmentsRequestDto> requestValidator,
+        IValidator<CreateAppointmentDto> createDtoValidator,
+        IValidator<UpdateAppointmentDto> updateDtoValidator) : IAppointmentService
```

Comment:
Move validation to validation middleware or use library method to apply request validation automatically